style.css

```diff
@@ -405,6 +405,62 @@ body {
   cursor: pointer;
 }
 
+/* The Modal (background) */
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: 5% auto;
+  padding: 20px 60px;
+  border: 1px solid #888;
+  width: 90%;
+  max-width: 800px;
+
+  /* text-align: center; */
+}
```

Comment:
- [ ] Please remember to delete the comment on line 430

- [ ] I suggest making the popup fit the mobile and desktop sizes properly without the need to scroll. 

- [ ] To match the Figma, the modal overlay need to have a **[blur](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur)**. 
![modal](https://user-images.githubusercontent.com/99656687/233322912-d36d6f94-4028-4794-8e49-bce1e6f2488d.png)